crates/matrix-sdk-crypto/src/identities/device.rs

```diff
@@ -928,6 +928,7 @@ pub(crate) mod tests {
     }
 
     #[test]
+    #[allow(clippy::redundant_clone)]
```

Comment:
@jplatte added this line to make Clippy happy even if it seems unrelated with this PR